api/db-cdr/changelog-schema/db.changelog-5.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="biopete" id="changelog-5">
+
+    <createTable tableName="achilles_results_concept">
+      <column name="id" type="BIGINT">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="analysis_id" type="BIGINT">
+        <constraints nullable="false"/>
+      </column>
+      <column name="stratum_1" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_1_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_2" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_2_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_3" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_3_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_4" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_4_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_5" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_5_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="count_value" type="BIGINT">
+        <constraints nullable="true"/>
+      </column>
+    </createTable>
+
+    <createTable tableName="achilles_results_dist_concept">
```

Comment:
Where is this table referenced?

Reply:
future used for box plot distribution  results. We don't have those yet but will.

Reply:
Let's add it in future when it's used?

Reply:
DB talked about .